x-pack/plugins/maps/public/components/map/mb/view.js

```diff
@@ -57,44 +118,25 @@ export class MBMapContainer extends React.Component {
     }
     if (this._mbMap) {
       this._mbMap.remove();
+      this._mbMap.off('mousemove', this._updateTooltipState);
```

Comment:
Not sure we need this. `remove` should clean up all listeners for use plus docs say "Use this method when you are done using the map and wish to ensure that it no longer consumes browser resources. Afterwards, you must not call any other methods on the map."